### PR TITLE
New: implement rule "no-console"

### PR DESCRIPTION
**Pull request by g-plane**

### CODE REVIEW

File: lib/rules/no-console.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * @author Pig Fang <g-plane@hotmail.com>
+ */
+"use strict";
+
+const ruleComposer = require("eslint-rule-composer");
+const utils = require("../utils");
+
+const rule = utils.getFixableRule("no-console");
+
+rule.meta.fixable = "code";
```
Comment (aladdin-add) [inline lib/rules/no-console.js]: nitpick: it has been assigned here: https://github.com/aladdin-add/eslint-plugin-autofix/blob/master/lib/utils.js#L12

File: tests/lib/rules/no-console.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            output: "if (true) {}",
+            errors: [{ messageId: "unexpected", type: "MemberExpression" }]
+        },
+        {
+            code: "if (true) { console.log }",
+            output: "if (true) {  }",
+            errors: [{ messageId: "unexpected", type: "MemberExpression" }]
+        },
+        {
+            code: "if (true) console.log()",
```
Comment (aladdin-add) [inline tests/lib/rules/no-console.js]: we might need to check the arguments:
```js
console.log(foo); // safe to remove
console.log(foo, bar): // safe to remove
console.log("foo"); // safe to remove
console.log(`foo`); // safe to remove

console.log(foo()); // unsafe to remove
console.log(`${foo()}`); unsafe to remove // unsafe to remove
```

btw, mind adding something like:
```js
let log = console.log;
```

Comment (g-plane) [reply tests/lib/rules/no-console.js]: So should we remove that when the arguments only contain literal or identifier?

Comment (aladdin-add) [reply tests/lib/rules/no-console.js]: I guess so, since we don't know if the others have side effects.

**Pull request merged**